accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/core/BasicMonotonicAccessioningWithAlternateRangesTest.java

```diff
@@ -67,8 +67,8 @@ public void testAlternateRangesWithDifferentGenerators() throws AccessionCouldNo
         List<AccessionWrapper<TestModel, String, Long>> evaAccessions = getAccessioningService(categoryId, INSTANCE_ID)
                 .getOrCreate(getObjectsForAccessionsInRange(1, 10));
         assertEquals(9, evaAccessions.size());
-        assertEquals(1, evaAccessions.get(0).getAccession().longValue());
-        assertEquals(9, evaAccessions.get(8).getAccession().longValue());
+        assertEquals(0, evaAccessions.get(0).getAccession().longValue());
```

Comment:
ok, I saw you changed the starting value in the properties, nevermind. this is closer to our use actually.

Reply:
Based on this question and mine below, I think a lot more comments need to be added to this method. So instead of simply

> New Block still not interleaved

list also the start and end values expected in that block, and what is the reason for that (including the calculations if necessary).